ably/state.go

```diff
@@ -29,7 +29,7 @@ func (st StateType) String() string {
 	case StateChan:
 		return "channel"
 	default:
-		return ""
+		return "invalid"
```

Comment:
I would have said that in that case, it should be logged instead as a warning or an error. We tend to do that a lot in other client libraries.

The thing is that we have not a proper logger at this point yet. Unless I missed it.